packages/code-du-travail-frontend/pages/integration/[slug].tsx

```diff
@@ -38,6 +38,12 @@ const IntegrationPage = ({ slug, hostname, protocol }): JSX.Element => {
 
 export const getServerSideProps = async ({ query, req }) => {
   const slug: string = query.slug;
+  const keys = Object.keys(integrationData);
```

Comment:
tu peux juste rajouter une url dans le test : code-du-travail-numerique/packages/code-du-travail-frontend/cypress/integration/not-found.spec.ts